assemblyline_core/metrics/heartbeat_formatter.py

```diff
@@ -115,13 +113,13 @@ def send_heartbeat(self, m_type, m_name, m_data, instances):
                     "msg": {
                         "inflight": {
                             "max": self.config.core.dispatcher.max_inflight,
-                            "outstanding": self.dispatch_active_hash.length()
+                            "outstanding": self.dispatch_active_hash.length(),
                         },
                         "instances": instances,
                         "metrics": m_data,
                         "queues": {
                             "ingest": self.dispatcher_submission_queue.length(),
-                            "files": self.dispatcher_file_queue.length(),
+                            "files": 0
```

Comment:
We should just get rid of that and have a patch in the UI to stop monitoring it.